node/src/components/small_network/symmetry.rs

```diff
@@ -173,8 +168,13 @@ impl ConnectionSymmetry {
         }
     }
 
+    #[cfg(test)]
     /// Indicates whether or not a connection should be cleaned up.
-    pub(super) fn should_be_reaped(&self, now: Instant, max_time_asymmetric: Duration) -> bool {
+    pub(super) fn should_be_reaped(
+        &self,
+        now: Instant,
+        max_time_asymmetric: std::time::Duration,
```

Comment:
Was the change intentional here? I have been asked to change it from qualified to unqualified in the past and `Duration` is fairly unambiguous, I think. :)

Reply:
Good point - `std::time::Duration` should be imported behind a `#[cfg(test)]` gate.

Reply:
Fixed in d9ae2b730b2572b58d76b49b5ce201f7b0dd1719

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/1843/commits/dc2919bc75167f006ab5842afbdeb288f311f845